catalog/spatial/wfs/1.0.0/spatial-wfs-v1_0_0-converter/src/test/java/org/codice/ddf/spatial/ogc/wfs/v1_0_0/catalog/converter/impl/TestGenericFeatureConverter.java

```diff
@@ -143,14 +145,14 @@ public void testUnmarshalSingleFeatureXmlToObject() {
                 mc.getAttribute(PROPERTY_PREFIX + WIDTH_ELEMENT)
                         .getValue());
 
-        assertEquals(getLocation(), mc.getLocation());
+        assertEquals(getLocation(), mc.getAttribute(Core.LOCATION).getValue());
         assertEquals(mc.getLocation(),
                 mc.getAttribute(PROPERTY_PREFIX + GROUND_GEOM_ELEMENT)
                         .getValue());
 
-        assertNotNull(mc.getCreatedDate());
-        assertNotNull(mc.getEffectiveDate());
```

Comment:
Since we're going to leave the `effective` logic in, could we please leave the test coverage for it in as well?